harvard-lts/Harvard-Geospatial-Library
# Local db migrate

**Remove Localdb migrate sh from Dockerfile**
* * *

# What does this Pull Request do?
We dont actually need the localdb script outside of local, so moving that to the DockerfileLocal and removing from Dockerfile

# How should this be tested?

You can pull this branch, spin up locally, and the script should be needed. If you then use the Dockerfile that script won't be used (docker ps will show you the commands and entry points)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
